packages/remix-dev/__tests__/fixtures/cloudflare/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
```

Comment:
Brought over the `node`/`deno`/`cloudflare` fixtures for the remix-dev tests, but if adapters live elsewhere dunno if these all make sense in here long term (at least deno + cloudflare)?